packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -173,6 +190,8 @@ export abstract class BaseService extends Resource
         maximumPercent: props.maxHealthyPercent || 200,
         minimumHealthyPercent: props.minHealthyPercent === undefined ? 50 : props.minHealthyPercent
       },
+      propagateTags: props.propagateTags,
```

Comment:
What would happen if you deployed to CFN with NONE in the place of this value?

Reply:
"PropagateTags" this will not show up then

Reply:
If I pass props.propagateTags with the value PropagatedTagSource.NONE, what value is passed to PropagateTags?

Reply:
undefined

Reply:
How does the baseservice know this?